package.json

```diff
@@ -124,22 +124,22 @@
     "@editorjs/code": "2.7.0",
     "@editorjs/inline-code": "1.3.1",
     "@editorjs/marker": "1.2.2",
-    "@nrwl/angular": "13.10.5",
-    "@nrwl/cli": "13.10.5",
-    "@nrwl/eslint-plugin-nx": "13.10.5",
-    "@nrwl/jest": "13.10.5",
-    "@nrwl/linter": "13.10.5",
-    "@nrwl/node": "13.10.5",
-    "@nrwl/nx-plugin": "13.10.5",
-    "@nrwl/storybook": "13.10.5",
+    "@nrwl/angular": "14.4.2",
+    "@nrwl/cli": "14.4.2",
+    "@nrwl/eslint-plugin-nx": "14.4.2",
+    "@nrwl/jest": "14.4.2",
+    "@nrwl/linter": "14.4.2",
+    "@nrwl/node": "14.4.2",
+    "@nrwl/nx-plugin": "14.4.2",
+    "@nrwl/storybook": "14.4.2",
```

Comment:
I think u are anticipating the lib we will have with angular 14

Reply:
as we discussed we will use 13 version with angular 13